.github/workflows/integration-informing.yaml

```diff
@@ -1,4 +1,4 @@
-name: Integration Tests (Strict)
+name: Informing Integration Tests
```

Comment:
What does 'Informating integration tests' mean? I think this name should be changed